app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.se.cyrcle.model.user
+
+data class User(
+    val userId: String,
+    val username: String = "",
+    val firstName: String? = null,
```

Comment:
I think we should replace `String? = null` by `String = ""` for the other inputs for consistency. We do not expect any other type besides `String` and except for the username they can be empty if the user doesn't want to give corresponding info.